src/style.css

```diff
@@ -121,6 +121,10 @@ input {
   justify-content: left;
```

Comment:
- [OPTIONAL] I suggest aligning your  list to the left to make it look more like the Figma design and giving the right padding and margins the "Today's todo" list :recycle: 

![Screenshot from 2023-04-08 19-32-20](https://user-images.githubusercontent.com/97834160/230735139-0e2df73c-368e-4efe-9040-10a6beb7f22c.png)